app/controllers/api/v1/doctors_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::V1::DoctorsController < ApplicationController
+  def index
+    @doctors = Doctor.all
+    render json: @doctors
+  end
+
+  def show
+    @doctor = Doctor.find(params[:id])
+    render json: @doctor
+  end
+
+  def create
+    @doctor = Doctor.new(doctor_params)
+    if @doctor.save
+      render json: 'Doctor created successfully'
```

Comment:
you should implement this way like this [link](https://web-crunch.com/posts/create-a-basic-api-with-ruby-on-rails), you should implement like the next text
```
def create 
  @band = Band.new(band_params)
  if @band.save
        render json: @band, status: :created, location: @band
      else
        render json: @band.errors, status: :unprocessable_entity
      end
end
```

Reply:
I do not think there is a problem with this implementation